app/Data/Data.php

```
@@ -0,0 +1,69 @@
+    }
+
+    public function toArray(): array
+    {
+        $array = parent::toArray();
+        if ($this->isFractal) {
+            $resourceKey = method_exists($this, 'getResourceName') 
+                ? $this->getResourceName() 
+                : strtolower(class_basename($this));
+
+            $additional = $this->getAdditionalData();
+            foreach ($additional as $key => $value) {
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)